### ISSUE TITLE
Scripts for figure 2

### ISSUE THREAD

**Issue by annerilotter** [yilinZhang-bio/Watermelon-pangenome#2]
Hi, I am missing the scripts for creating the plots in figure 2 and the scripts to generate the relevant data. I am missing the same for parts d and e of figure 3. Could they please be added or shared

**Comment by yilinZhang-bio** [comment]
I apologize for the delayed response. I've been traveling and just got the chance to address your question.


For Figure 2:
The data for this figure was generated using OrthoFinder with the following command:

orthofinder -f Data -t 128 -S diamond -M msa -T fasttree

After running this command, you can use the statistics from the Comparative_Genomics_Statistics output and the information from the Orthogroups to create Figure 2. The specific plots were generated based on these results, without requiring additional scripts.

For Figure 3 (parts d and e):
These parts were also created based on statistical analysis of the results. We compiled the relevant data into an Excel sheet, which was then used to create the plots using ggplot2 in R.



Please let me know if you need any further clarification or if you'd like more specific details on any part of the process.

**Comment by annerilotter** [comment]
Thank you for the reply. Did you use bedtools intersect for figure 3e?

**Comment by yilinZhang-bio** [comment]
Of course. We did use bedtools intersect for generating the results shown in Figure 3e. Specifically, we used parameters like bedtools intersect -a <fileA> -b <fileB> -wao. It's important to note that there can be multiple overlapping regions. We calculated the overlaps based on the following priority: exons > introns > upstream of genes > downstream of genes > intergenic regions. 
I hope this helps!








发件人：annerilotter ***@***.***>
发送日期：2024-07-24 16:03:57
收件人：yilinZhang-bio/Watermelon-pangenome ***@***.***>
抄送人：yilinZhang-bio ***@***.***>,Comment ***@***.***>
主题：Re: [yilinZhang-bio/Watermelon-pangenome] Scripts for figure 2 (Issue #2)

Thank you for the reply. Did you use bedtools intersect for figure 3e?

**Comment by yilinZhang-bio** [comment]
Of course. We did use bedtools intersect for generating the results shown in Figure 3e. Specifically, we used parameters like bedtools intersect -a <fileA> -b <fileB> -wao. It's important to note that there can be multiple overlapping regions. We calculated the overlaps based on the following priority: exons > introns > upstream of genes > downstream of genes > intergenic regions. 
I hope this helps!

**Comment by yilinZhang-bio** [comment]
Of course. We did use bedtools intersect for generating the results shown in Figure 3e. Specifically, we used parameters like bedtools intersect -a <fileA> -b <fileB> -wao. It's important to note that there can be multiple overlapping regions. We calculated the overlaps based on the following priority: exons > introns > upstream of genes > downstream of genes > intergenic regions. 
I hope this helps!

**Comment by annerilotter** [comment]
Great. How does the upstream/downstream of genes work exactly as they are obviously not a feature of the annotation file?

**Comment by yilinZhang-bio** [comment]
2000 bp is our standard
